src/main/java/Task.java

```diff
@@ -0,0 +1,32 @@
+public class Task {
+    protected static String[] tasks = new String[100];
+    protected static boolean[] tasksCompleted = new boolean[100];
```

Comment:
Please avoid the use of Magic Numbers. Instead declare a constant with the value you deem is right, and use that constant.

Or better, you may choose to use `ArrayList` which dynamically resizes as per need, and you may not require such constants. However, everything comes at a tradeoff, so you may do some research into `Array` vs `ArrayList` and choose what fits the best for your needs.

Reference: https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers